src/apis/BackHandler.re

```diff
@@ -1,14 +1,14 @@
 type remove = {. [@bs.meth] "remove": unit => unit};
 
 [@bs.scope "BackHandler"] [@bs.module "react-native"]
-external exitApp: unit => unit = "";
+external exitApp: unit => unit = "exitApp";
```

Comment:
similar update in .md file is missing here & in all others apis below (but maybe: should we care? no big deal)

Reply:
Done